### PR TITLE
Lots of changes/improvements to get us ready for initial MVP Launch

### PR DESCRIPTION
**Pull request by WafflesMcDuff**
### Added

- #13 
* Player Portal 
    - #12 
* Admin Portal 
    - #10 
    - #9 
        TODO: Display in the admin portal before Issue 9 is resolved.
    - #8 
    - #7 

### Fixed

- #14 

### Changed

* Player portal
    - #11

### CODE REVIEW

File: admin/index.php [hunk 1]
```diff
@@ -46,11 +54,24 @@
...
 
+if ($current_event == "Yes") {
+	$current_event_characters = $api->get_characters_upcoming_event();
+}
+
+function hasId($arr, $id)
+{
+	foreach ($arr as $value) {
+		if ($value['characterID'] == $id)
+			return true;
```
Comment (mrperson123) [inline admin/index.php:65]: Missing the brackets

File: admin/index.php [hunk 2]
```diff
@@ -155,56 +183,89 @@
...
+						</label>
+						<h4>
+							<?php
+							if (isset($_GET['current_event']) && $_GET['current_event'] == "Yes") {
+								echo 'This Event Only';
+							} else {
+								echo 'All active characters';
+							}
+							?>
+						</h4>
```
Comment (mrperson123) [inline admin/index.php:210]: You can't just set an h4 level out of nowhere. This is not a11y friendly. They need to be nested

File: admin/index.php [hunk 3]
```diff
@@ -155,56 +183,89 @@
...
-				Admin - Concept/Backstory editor
-				<?php
-				if ($faction != "")  echo ' - ' . $faction . ' only';
-				?>
-			</h1>
+					Admin - Concept/Backstory editor
+					<?php
+					if ($faction != "")
+						echo ' - ' . $faction . ' only';
+					?>
```
Comment (mrperson123) [inline admin/index.php:219]: Missing the brackets

File: admin/index.php [hunk 4]
```diff
@@ -155,56 +183,89 @@
...
-				<div class="tabs">
-					<div data-tab="concept"
-						class="tab<?php if ( $tab === 'concept' )
-					echo ' active' ?>">
+				<button data-tab="concept" <?php if ($tab === 'concept') echo 'class="active"' ?> onclick="window.location.href='?tab=concept&faction=<?php echo $faction; ?> . '&current_event=' . $current_event';">Concept</button>
+				<button data-tab="backstory" <?php if ($tab === 'backstory') echo 'class="active"' ?> onclick="window.location.href='?tab=backstory&faction=<?php echo $faction . '&current_event=' . $current_event; ?>';">Backstory</button>
+				<button data-tab="completed" <?php if ($tab === 'completed')
+					echo 'class="active"' ?>onclick="window.location.href='?tab=completed&faction=<?php echo $faction . '&current_event=' . $current_event; ?>';">Completed</button>
+				<button data-tab="submit_existing" <?php if ($tab === 'submit_existing')
+					echo 'class="active"' ?>onclick="window.location.href='?tab=submit_existing&faction=<?php echo $faction . '&current_event=' . $current_event; ?>';">Submit
```
Comment (mrperson123) [inline admin/index.php:248]: Missing the {} brackets in the if statements

File: admin/index.php [hunk 5]
```diff
@@ -155,56 +183,89 @@
...
 						<h2>Backstory</h2>
 					<?php require './partials/backstory.php'; ?>
 				</div>
-				<div data-tab="completed" class="tab<?php if ( $tab === 'completed')	echo ' active' ?>">
+				<div data-tab="completed" class="tab<?php if ($tab === 'completed') echo ' active' ?>">
 						<h2>Completed</h2>
 					<?php require './partials/completed.php'; ?>
 				</div>
+				<div data-tab="submit_existing" class="tab<?php if ($tab === 'submit_existing')
+					echo ' active' ?>">
```
Comment (mrperson123) [inline admin/index.php:265]: Missing the brackets again

**Review by mrperson123 — CHANGES_REQUESTED** [review]

File: admin/index.php (hunk 2 above)
Comment (WafflesMcDuff) [reply admin/index.php]: @rolf-yoast  - What do you mean by nested? Are you saying that an H4 needs to be inside an H3, inside an H2, inside an H1?

**Review by mrperson123 — APPROVED** [review]

**Pull request merged**